### PR TITLE
[fix] minify

### PR DESCRIPTION
**Pull request by jcrugzz**
1. Make sure minify doesnt overwrite regular content
2. Default minify in test as well

### CODE REVIEW

**Comment by bmeck** [comment]
LGTM, +1 on moving to .find

File: test/lib/routes/index.test.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ describe('Application routes', function () {
       post.end(new Buffer(JSON.stringify(data)));
 
       app.construct.once('store', function store(spec, progress, resData) {
-        const file = resData.files.files[0];
+        const file = resData.files.files.find(file => file.filename === 'index.min.jsx');
```
Comment (indexzero) [inline test/lib/routes/index.test.js:143]: Is the `.jsx` here because the test fixture is `.jsx`?

**Review by indexzero — COMMENTED** [review]
We should drop `.jsx` from fixtures if possible as part of this work.

File: test/lib/routes/index.test.js (hunk 1 above)
Comment (jcrugzz) [reply test/lib/routes/index.test.js:143]: Yes. And sure we can but this is duplicate work as I have to do it in the workers-factory library.

**Comment by jcrugzz** [comment]
Handled JSX fix in https://github.com/warehouseai/workers-factory

**Pull request merged**